packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/pages/example-print-extension.doc.ts

```diff
@@ -0,0 +1,281 @@
+import type {LandingTemplateSchema} from '@shopify/generate-docs';
+
+const examplePath = '../examples/print-example';
+
+const data: LandingTemplateSchema = {
+  title: 'Build a print extension',
+  description:
+    'Learn how to build a POS print extension that generates, previews, and prints custom documents.',
+  id: 'example-print-extension',
+  image: '/assets/landing-pages/templated-apis/hero.png',
+  darkImage: '/assets/landing-pages/templated-apis/hero-dark.png',
+  tabletImage: '/assets/landing-pages/templated-apis/hero.png',
+  tabletDarkImage: '/assets/landing-pages/templated-apis/hero-dark.png',
+  mobileImage: '/assets/landing-pages/templated-apis/hero.png',
+  mobileDarkImage: '/assets/landing-pages/templated-apis/hero-dark.png',
+  sections: [
+    {
+      type: 'Generic',
+      anchorLink: 'intro',
+      title: 'Introduction',
+      sectionContent: `
+This tutorial shows you how to create a print extension that lets merchants generate and preview documents before printing.
+
+What you'll achieve:
+- Create a backend service that serves print-ready documents
+- Build a POS extension with preview and print capabilities
+- Implement error handling for a reliable printing experience
+- Test your extension in a development environment
+
+![Print Extension in Action](/assets/api/pos/pos-ui-extensions-print-example.gif)
+      `,
+      sectionCard: [
+        {
+          name: 'Print API Reference',
+          subtitle: 'Learn about the Print API',
+          url: '/docs/api/pos-ui-extensions/apis/print-api',
+          type: 'api',
+        },
+        {
+          name: 'PrintPreview Component',
+          subtitle: 'Implement document previews',
+          url: '/docs/api/pos-ui-extensions/components/printpreview',
+          type: 'component',
+        },
+      ],
+    },
+    {
+      type: 'Generic',
+      anchorLink: 'requirements',
+      title: 'Before you start',
+      sectionContent: `
+To make the most of this tutorial, you'll need:
+
+* Complete the [Getting started guide](/docs/api/pos-ui-extensions/getting-started)
+  - Partner account
+  - Development store with POS
+  - Latest version of Shopify CLI
+  - Basic app setup (access scopes only needed if accessing Shopifydata)
```

Comment:
```suggestion
  - Basic app setup (access scopes only needed if accessing Shopify data)
```

Reply:
I believe the getting-started guide covers all of these, right? Maybe this list is redundant and not necessary?

Reply:
I thought it would be valuable to highlight the important requirements of that guide / refresh what they need.